osf/models/preprint_provider.py

```diff
@@ -34,24 +33,34 @@ def __unicode__(self):
 
     @property
     def top_level_subjects(self):
```

Comment:
Added in https://github.com/CenterForOpenScience/osf.io/pull/7127/commits/a431d82ec0fe7ac1e8310766961ba86ea73ff9e5